utils/src/main/java/software/amazon/awssdk/utils/StringUtils.java

```diff
@@ -104,7 +104,7 @@
  * @see java.lang.String
  */
 @ReviewBeforeRelease("Remove the methods we don't end up using (and we've removed software.amazon.awssdk.core.util.StringUtils).")
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
Public

Reply:
Before GA, sure. It's not ready to be public right now, though.